DopplerJobsServer/Startup.cs

```diff
@@ -91,18 +91,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseStaticFiles();
 
-            if (env.IsDevelopment())
+            app.UseHangfireDashboard("/hangfire", new DashboardOptions
             {
-                app.UseHangfireDashboard("/hangfire", new DashboardOptions
-                {
-                    PrefixPath = !env.IsDevelopment() ? "/jobs" : null,
-                    Authorization = new[] {new HangfireAuthorizationFilter()}
-                });
-            }
+                PrefixPath = !env.IsDevelopment() ? "/jobs" : null,
```

Comment:
Mmm, I do not like it. Because `/jobs` path does not depend on this repository.

What happens if we decide to use a different path in a different deployment?

Maybe you can add a configuration like _base path_ and configure it in Swarm.

Reply:
Yes, I agree, It is for Hangfire Dashboard to work in production, maybe we can remove the Hangfire dashboard in prod,
if we change the path in different deployment,  it would not work

Reply:
What do you think about an intermediate solution that could be overridden if we need it: You can include this prefix into your configuration files (in this same repository)

Reply:
Yes Is a good idea, I'll do it